### PR TITLE
Explicitly set `hasColorScreen = no` for B&W devices

### PR DESCRIPTION
**Pull request by dmalinovsky**
See the discussion in https://github.com/koreader/koreader-base/pull/1752/files/279a9d9fdeb4ae108a98f635d451f5d3ab0ffcea#r1546370061.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/11608)

### CODE REVIEW

File: frontend/device/pocketbook/device.lua [hunk 1]
```diff
@@ -67,6 +67,9 @@ local PocketBook = Generic:extend{
     -- Will be set appropriately at init
     isB288SoC = no,
 
+    -- Most of the devices are B&W.
+    hasColorScreen = no,
+
```
Comment (ElimGarak1) [inline frontend/device/pocketbook/device.lua:72]: This idea came to me too. But no success. Previously, I even applied it to certain PocketBook devices that I used for testing.

**Comment by hius07** [comment]
I think the reason is
https://github.com/koreader/koreader/blob/42c93a7623c6d04a2001f943ddc5b0b0638e3a24/frontend/device/pocketbook/device.lua#L22
and
https://github.com/koreader/koreader/blob/42c93a7623c6d04a2001f943ddc5b0b0638e3a24/frontend/device/generic/device.lua#L68
that produces always truthy `if fb.device.hasColorScreen` in your framebuffer_pocketbook.lua.

So, the solution would be `if fb.device.hasColorScreen == "yes"`.

**Comment by dmalinovsky** [comment]
Good catch, @hius07.  Let me fix that.

**Comment by dmalinovsky** [comment]
@hius07, the solution will be a bit different, see https://github.com/koreader/koreader-base/pull/1757.

**Comment by hius07** [comment]
I see, yes/no are functions, good.

**Pull request closed without merging**